bin/install_actinia.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/sh
+#############################################################################
+#
+# Purpose: This script will install actinia_core (REST API for GRASS GIS 7)
+#
+# References:
+#          - Code: https://github.com/mundialis/actinia_core/
+#          - Publication: https://doi.org/10.5281/zenodo.2631917
+#          - Tutorial: https://actinia.mundialis.de/tutorial/
+#
+# Requirements: GRASS GIS 7, Python, redis
+#
+# actinia URL after installation: http://localhost:8080/api/v1/version
+#
+#################################################################################
+# Copyright (c) 2018-2019 Sören Gebbert and mundialis GmbH & Co. KG, Bonn.
+# Copyright (c) 2016-2019 The Open Source Geospatial Foundation and others.
+#
+# Installer script author: Markus Neteler <neteler mundialis.de>
+#
+# Licensed under the GNU LGPL version >= 2.1.
+#
+# This library is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 2.1 of the License,
+# or any later version.  This library is distributed in the hope that
+# it will be useful, but WITHOUT ANY WARRANTY, without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details, either
+# in the "LICENSE.LGPL.txt" file distributed with this software or at
+# web page "http://www.fsf.org/licenses/lgpl.html".
+#################################################################################
+
+# About:
+# =====
+# This script will install actinia_core with actinia_statistic_plugin
+#
+# Script inspired by https://github.com/mundialis/actinia_core/blob/master/docker/actinia-core/Dockerfile
+#
+# This does not attempt to install GRASS GIS, that is done in install_grass.sh.
+#################################################################################
+
+./diskspace_probe.sh "`basename $0`" begin
+BUILD_DIR=`pwd`
+####
+
+# live disc's username is "user"
+if [ -z "$USER_NAME" ] ; then
+   USER_NAME="user"
+fi
+USER_HOME="/home/$USER_NAME"
+
+TMP_DIR=/tmp/build_actinia
+mkdir "$TMP_DIR"
+
+apt-get --quiet update
+
+# get source code
+# https://github.com/mundialis/actinia_core
+cd "$TMP_DIR"
+git clone --single-branch https://github.com/mundialis/actinia_core.git
+cd actinia_core/
+
+# generate virtual env for Python3
+apt -y install virtualenv
+apt -y install python3-dev
+virtualenv -p python3 venv
```

Comment:
virtualenv, pip and conda are not permitted/used in OSGeoLive. For python software we need to have debian packages to install from.

Reply:
The replacement of the `pip install -r requirements.txt` call may look like this:

```
apt -y install python3-boto3 python3-docutils python3-flask python3-flask-restful \
    python3-geopandas gunicorn python3-numpy python3-pandas python3-passlib python3-ply \
    python3-magic python3-shapely python3-sphinx redis python3-requests uwsgi
```

Here I am unsure if ok as no "python3-xxx" available:
```
# no py3? or ok?
apt -y install python-flask-httpauth python-scikits-learn
```

And here I am not aware of Ubuntu packages, how to install?  (I am more familiar with the RPM world...)
```
# flask-restful-swagger-2 flask-cors fluent-logger google-cloud rq
```

Reply:
ok we can create the missing dependencies, no worries.

Reply:
To avoid the virtualenv, we could hardcode "python3" in this script.

Reply:
it is not only virtualenv, we avoid pip too.

Reply:
actinia and dependencies now have a debian package:

https://launchpad.net/~gcpp-kalxas/+archive/ubuntu/osgeolive/+packages

Reply:
Actually the package python3-actinia-core will pull this list of dependencies automatically through apt.